packages/@o3r/artifactory-tools/README.md

```diff
@@ -3,122 +3,154 @@
-  -h, --help                           Output usage information
+The required options include:
+* the JFrog artifactory URL (`--artifactory-url <artifactoryUrl>`)
+* a method of authentication (`--api-key <apiKey>` or `--basicAuth <base64>`)
+* the artifact repository to clean up (`--repository <repository>`)
+* the artifact paths to clean up (`--path <path>`)
+
+### Options
+
+| Option                                                   | Alias  | Value Type | Default Value | Description                                                                                                                                          |
+|----------------------------------------------------------|:------:|------------|---------------|------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `--artifactory-url <artifactoryUrl>` <br> **(Required)** |  `-u`  | `string`   |               | JFrog artifactory URL                                                                                                                                |
+| `--path <path>` <br> **(Required)**                      |  `-p`  | `string`   |               | Artifact paths to clean up (using matcher from AQL language). Be careful not to include release artifacts in the path. <br> ex: `sample/path/*-PR-*` |
+| `--repository <repository>` <br> **(Required)**          |  `-r`  | `string`   |               | Artifact repository to clean up                                                                                                                      |
+| `--api-key <apiKey>`                                     |  `-a`  | `string`   |               | Artifactory API key of the user that can be used to log in                                                                                           |
+| `--basicAuth <base64>`                                   |  `-b`  | `string`   |               | Base64 encoding of username:password (password already encrypted from artifactory UI)                                                                |
+| `--dry-run <dryRun>`                                     |        | `boolean`  | `false`       | List all files that would be deleted without actually deleting them                                                                                  |
+| `--duration-kept <durationKept>`                         |  `-d`  | `number`   | `1`           | All artifacts older than this value (in days) will be deleted                                                                                        |
+| `--pr-versions <prVersions>`                             | `-pr`  | `number`   | `1`           | Number of PR versions that will be kept                                                                                                              |
```

Comment:
Is `-pr` working? it is not considered as `-p -r`?

Reply:
I tested and `-pr` is working yes, but I can change the alias to a single letter if you prefer (like `-n` for example since we want the number of PR versions to keep)

Reply:
I think changing the alias would be actually safer